src/workerd/api/hibernatable-web-socket.h

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2017-2022 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+
+#pragma once
+
+#include <kj/debug.h>
+
+#include <workerd/io/worker-interface.capnp.h>
+#include <workerd/api/global-scope.h>
+#include <workerd/io/worker-interface.h>
+
+namespace workerd::api {
+
+class HibernatableWebSocketEvent final: public ExtendableEvent {
+public:
+  explicit HibernatableWebSocketEvent();
+
+  static jsg::Ref<HibernatableWebSocketEvent> constructor(kj::String type) = delete;
+
+  // TODO(now): return correct ws instead of the current stub implementation
```

Comment:
Probably not needed anymore? Could just add a `TODO(soon)` to the comments in relevant methods in `hibernatable-web-socket.c++`